cachito/web/app.py

```diff
@@ -68,6 +70,17 @@ def create_app(config_obj=None):
     else:
         load_config(app)
 
+    # Configure logging
+    default_handler.setFormatter(
+        logging.Formatter(fmt=app.config['CACHITO_LOG_FORMAT'], datefmt='%Y-%m-%d %H:%M:%S')
+    )
+    app.logger.setLevel(app.config['CACHITO_LOG_LEVEL'])
+    for logger_name in app.config['CACHITO_ADDITIONAL_LOGGERS']:
+        logger = logging.getLogger(logger_name)
```

Comment:
Would this be for setting the logging level of third party libraries?
I think that's pretty neat.

Reply:
Yes, exactly!